api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -373,4 +373,8 @@ public static CustomResourceDefinition kafkaRebalance() {
             throw new RuntimeException(e);
         }
     }
+
+    public static int getCrdsCount() {
```

Comment:
I have to say that somehow the `getCrdsCount` seems weird. Wouldn't `getCrdCount` be more natural? WDYT @samuel-hawker?

Reply:
`getNumCrds()`

Reply:
See whilst I was reviewing this, I thought the same, but then went through the class and realized that it has some a large number of functions that being as explicit as getCrdsCount was fine, I do agree though with Tom's suggestion getNumCrds is good too.

Reply:
I might add that 
`getKnownNumCrds` may be marginally better as it is more accurate, (the crds known to the class), as opposed to say number of Crds total, but a javadoc comment would be fine for clarifying this, I am happy with it as you have changed it..